### DISCUSSION TITLE
Simple RSS task produces 404 client error for select entries.

### DISCUSSION THREAD

**Discussion by xusiai** [Flexget/Flexget#3838] [Q&A] [answered]
Hi everyone, 

I have a consistent log error which I don't understand and therefore hoping someone here can solve it. 

I have a simple rss task: 

```
  download-mv:
    rss:
      url: "{? feeds.mv ?}"
    accept_all: yes
    transmission:
      <<: *transmission
```
This runs and adds *some* entries to Transmission. However it also produces an error for several entries which are *not* added to Transmission (but should be and are from the same feed). Here is an example:  

```
-mv Failed Asteroid City (2023) (1080p BluRay x265 HEVC 10bit AAC 7.1 Tigole) [QxR] (Network error during request: 404 Client Error: Not Found for url: http://x.x.x.x:9117/dl/1337x/?jackett_apikey=<apikey>&path=Q2ZESjhBZGt5VDVkenZKTmdoVzJDMHhqOFFnZmVMZk1FVDlWNV9lZUlwem8zMHpseXZWQU1FMlh0Rk54RnJGUXdVYjNzcWlZZmtvOHRXYzgzMkw0a2hFMmI4eWRDMkxicmY0S1dseU5uMmJOZ2Z4RTVzRl9JMjBUZDRXeHRybjBOeHRBUGRTRDNDb3c0SXhYYThtd1ZXX1dCVkVSNXBDc0VBenBPT1hBcC1xRV9hcDd2bV9lNUlmT2NVTFRBVGI1YWdwa1k5ZXRRenJDbXdOTVZmVno3bGNuNFU5MGh5engxaVVTMWdHanFKSWl0em5JMDhSbTdobi1RWDh6WkpfczVZLS1CZw&file=Asteroid+City+(2023)+(1080p+BluRay+x265+HEVC+10bit+AAC+7.1+Tigole)+%5BQxR%5D)
```

Since the task seem to fetch the feed and adds *some* entries it doesn't seem to be a network or Jackett issue, so any idea what's causing this?

**Comment by gazpachoking** [comment]
It seems like it got a 404 when it tried to download that particular torrent. Seems like probably an issue with your rss feed, jacket, or the tracker

**Reply by xusiai** [reply, marked as answer]
Thanks. After some digging I think what seems to happen is the torrent file download fails. Jackett is supposed to fall back to magnets but does not seem to do so. Changing the primary method to fetching magnets instead of files in Jackett seems to have solved it.
